pkg/api/context/models.go

```diff
@@ -77,8 +77,7 @@ func (models Models) GetTrainingDatasets() TrainingDatasets {
 	return trainingDatasets
 }
 
-func ValidateModelTargetType(targetDataTypeStr string, modelType string) error {
-	targetType := userconfig.ColumnTypeFromString(targetDataTypeStr)
+func ValidateModelTargetType(targetType userconfig.ColumnType, modelType string) error {
```

Comment:
`modelType` another candidate for enums

Reply:
Agreed. We can do that in a separate commit (I also noticed the same can be done for csv/parquet in environments config)